libr/core/meson.build

```diff
@@ -52,6 +52,7 @@ files = [
   'project.c',
   'pseudo.c',
   'rtr.c',
+  'r_core_rabin2.c',
```

Comment:
i would name this main_rabin2.c instead

this way we will have all bin programs prefixed with 'main' so its easy to find them and matches the function name